service/jobs/ext_api_test.go

```diff
@@ -0,0 +1,299 @@
+package jobs
+
+import (
+	"context"
+	"testing"
+
+	"github.com/databricks/databricks-sdk-go/qa"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func commonFixtureWithStatusResponse() qa.HTTPFixtures {
```

Comment:
IMO the common fixture like this is less readable. Each test depends only on one or two request mocks. In that case, I think they should be next to the test itself. Otherwise, the reader needs to bounce back and forth between the test and this definition. For example:
```go
func Test...() {
    t.Run("my test", func() {
        mock := []qa.HTTPFixture{...}
        client, server := mock.Client(t)
        ...
    })
}
```